homeassistant/components/ddwrt/device_tracker.py

```diff
@@ -116,28 +116,34 @@ def _update_info(self):
         """
         _LOGGER.debug("Checking ARP")
 
-        endpoint = "Wireless" if self.wireless_only else "Lan"
-        url = f"{self.protocol}://{self.host}/Status_{endpoint}.live.asp"
-        data = self.get_ddwrt_data(url)
+        url = f"{self.protocol}://{self.host}/Status_Wireless.live.asp"
```

Comment:
Please extract the device specific interface code and data parsing to a standalone library published on PyPI.

https://developers.home-assistant.io/docs/creating_component_code_review#4-communication-with-devicesservices

Reply:
I think that's kind of ridiculous as "the code" is just fetching one or two JSON-like data structures and parsing those for MAC addresses.
If anyone wants to add the overhead of creating a 3rd party library for that, feel free to do so.
I was just trying to help by fixing a bug in what's not even my code.
Well, I guess it's not going to happen then.